src/content/apis/translations/yoda-translations.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Yoda translator API
+description: Yoda translator API gives you API access to yoda translator, so that you can integrate this functionality in your websites or applications.
```

Comment:
Update the description to be of generic type as mentioned above

Reply:
done